plugins/FrameCombiner/src/main/java/org/micromanager/plugins/framecombiner/FrameCombinerConfigurator.java

```diff
@@ -150,16 +163,28 @@ public PropertyMap getSettings() {
 
       // Save preferences now.
       setProcessorAglo((String) processorAlgoBox_.getSelectedItem());
+      setProcessorDimension((String) processorDimensionBox_.getSelectedItem());
       setNumerOfImagesToProcess(Integer.parseInt(numerOfImagesToProcessField_.getText()));
       setChannelsToAvoid(channelsToAvoidField_.getText());
 
       PropertyMap.PropertyMapBuilder builder = studio_.data().getPropertyMapBuilder();
+      builder.putString("processorDimension", (String) processorDimensionBox_.getSelectedItem());
       builder.putString("processorAlgo", (String) processorAlgoBox_.getSelectedItem());
       builder.putInt("numerOfImagesToProcess", Integer.parseInt(numerOfImagesToProcessField_.getText()));
       builder.putString("channelsToAvoid", channelsToAvoidField_.getText());
       return builder.build();
    }
 
+    private String getProcessorDimension() {
+        return studio_.profile().getString(FrameCombinerConfigurator.class,
+                PROCESSOR_DIMENSION, FrameCombinerPlugin.PROCESSOR_DIMENSION_Z);
+    }
+
+    private void setProcessorDimension(String processorDimension) {
+        studio_.profile().setString(FrameCombinerConfigurator.class,
+                PROCESSOR_ALGO, processorDimension);
```

Comment:
this is an Oops. I didn't saw it. You're right, it should be PROCESSOR_DIMENSION